admin/admin-service/src/main/java/io/mosip/admin/packetstatusupdater/util/EventEnum.java

```diff
@@ -76,6 +76,9 @@ public enum EventEnum {
 			"successfully return the applicantPhoto and dob", "ADM-AVD", "admin service", "NO_ID", "NO_ID_TYPE",
 			AuditConstant.APPLICATION_ID, AuditConstant.APPLICATION_NAME),
 	APPLICANT_VERIFICATION_ERROR("ADM-AVD-503",AuditConstant.AUDIT_SYSTEM,"Request for Applicant Verification","Failed to call Rest api - %s","ADM-AVD","admin service","NO_ID","NO_ID_TYPE",AuditConstant.APPLICATION_ID,AuditConstant.APPLICATION_NAME),
+	APPLICANT_RID_NOT_FOUND("ADM-AVD-509",AuditConstant.AUDIT_SYSTEM,"Request for Applicant Verification","RID not found","ADM-AVD","admin service","NO_ID","NO_ID_TYPE",AuditConstant.APPLICATION_ID,AuditConstant.APPLICATION_NAME),
```

Comment:
are these error codes added in the artifactory i18n files

Reply:
Error codes are not added for the different languages in artifactory i18n files

Reply:
Added the errorCode in artifictory i18n files:https://github.com/mosip/artifactory-ref-impl/pull/455/commits/089f981679c616d8af5e767830c4bd34e068f070